pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -118,7 +118,7 @@ let
         ] ++ optionals x11Support [ tcl tk xlibsWrapper libX11 ]
     )
     ++ optional zlibSupport zlib
-    ++ optional stdenv.isDarwin CF;
+    ++ optionals stdenv.isDarwin [ coreutils CF ];
```

Comment:
Why do you need `coreutils` in `buildInputs` as well as the literal full path above?